### DISCUSSION TITLE
Signac into ArchR

### DISCUSSION THREAD

**Discussion by cserger** [swaruplabUCI/ArchRtoSignac#16] [Q&A]
Hello, thanks for developing this conversion package. :) 

Is there any way to do the opposite? 
Converting an fully processed Signac object into an ArchR objects (keeping the UMAP, QCs etc)? 

Thanks a lot, 
Clara

**Comment by rootze** [comment]
> Hello, thanks for developing this conversion package. :)
> 
> Is there any way to do the opposite? Converting an fully processed Signac object into an ArchR objects (keeping the UMAP, QCs etc)?
> 
> Thanks a lot, Clara

Hello Clara, 
Currently, we don't have the function to do that.

**Comment by rootze** [comment]
@cserger ArchRproject is a bit hard to deal with, as it is not only saved as an ArchRProject, but each sequencing sample has its own arrow file. If you don't mind me asking, what functions do you need from ArchR specifically? Maybe we can find an alternative method for you.

**Reply by cserger** [reply]
Hello, thanks for your answer. In general I would just like to use their pseudotime analysis and TF motif detection along this pseudotime. But thank you, I will find a way. :)
